leetcode/math.md

```diff
@@ -0,0 +1,153 @@
+# math
```

Comment:
WRONG GROUP in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message